user_guide_src/source/libraries/time.rst

```diff
@@ -203,17 +203,17 @@ The exact time displayed is determined in the following manner:
 =============================== =================================
 Time difference                  Result
 =============================== =================================
-$time > 1 year && < 2 years      in 1 year / 1 year ago
-$time > 1 month && < 1 year      in 6 months / 6 months ago
-$time > 7 days && < 1 month      in 3 weeks / 3 weeks ago
-$time > today && < 7 days        in 4 days / 4 days ago
-$time == tomorrow / yesterday    Tomorrow / Yesterday
-$time > 59 minutes && < 1 day    in 2 hours / 2 hours ago
-$time > now && < 1 hour          in 35 minutes / 35 minutes ago
+1 year < $time < 2 years         in 1 year / 1 year ago
+1 month < $time < 1 year         in 6 months / 6 months ago
+7 days < $time < 1 month         in 3 weeks / 3 weeks ago
+today < $time < 7 days           in 4 days / 4 days ago
+$time == yesterday / tomorrow    Yesterday / Tomorrow
+59 minutes < $time < 1 day       in 2 hours / 2 hours ago
+now < $time < 1 hour             in 35 minutes / 35 minutes ago
 $time == now                     Now
 =============================== =================================
 
-The exact language used is controlled through the language file, **Time.php**.
+The result strings are coming from the language file, **app/Language/{locale}/Time.php**.
```

Comment:
By default, there is no `app/Language/{locale}/Time.php`.

Reply:
@kenjis Yes. There is one under system/Language/en/Time.php. How do you think we could change this sentence? Because the current sentence in the guide is more incomprehensible.

Reply:
How about this?

The result strings are coming from the system language file **Time.php**. 
If you want to overwrite them, create **app/Language/{locale}/Time.php**.